source/NVDAObjects/UIA/__init__.py

```diff
@@ -106,21 +172,37 @@ def _get_NVDAObjectAtStart(self):
 	def _get_bookmark(self):
 		return self.copy()
 
-	def _getControlFieldForObject(self, obj):
+	def _getControlFieldForObject(self, obj,isEmbedded=False,startOfNode=False,endOfNode=False):
+		"""
+		Fetch control field information for the given UIA NVDAObject.
+		@ param obj: the NVDAObject the control field is for.
+		@type obj: L{UIA}
+		@param isEmbedded: True if this NVDAObject is for a leaf node (has no useful children).

```

Comment:
Unless I'm missing something, this method doesn't seem to use the `isEmbedded` argument. Is this still needed/is it likely to become necessary in the near-term?